tools/pagerduty-migrator/migrator/oncall_api_client.py

```diff
@@ -21,6 +21,12 @@ def api_call(method: str, path: str, **kwargs) -> requests.Response:
             cooldown_seconds = int(e.response.headers["Retry-After"])
             sleep(cooldown_seconds)
             return api_call(method, path, **kwargs)
+        elif e.response.status_code == 400:
```

Comment:
Good idea to add some exception logging! Slight improvement could be
```suggestion
        else:
            print(TAB + ">>> " + response.json()["detail"])
            raise
```
to log any unhandled exceptions and raise further.